src/frontend/komponenter/ManuellJournalfør/KnyttJournalpostTilBehandling.tsx

```diff
@@ -137,7 +139,8 @@ export const KnyttJournalpostTilBehandling: React.FC = () => {
                                         }}
                                     />
                                 </StyledTd>
-                                <td>{behandling.status}</td>
+                                <td>{formaterTilKunFørstBokstavStor(behandling.status)}</td>
```

Comment:
You can use .capitalize() for this I think 👍

Reply:
What can .capitalize() do? I searched but did not find the function. I need to capitalize the first letter of ALL words in a name, and is the function capable of doing it?